src/query/frontend/stripped_lexer_constants.hpp

```diff
@@ -145,6 +145,7 @@ const trie::Trie kKeywords = {"union",
                               "drop",
                               "show",
                               "stats",
+                              "status",
```

Comment:
I think we forgot before to add keywords for `LOCK`, `UNLOCK`, `DATA`, and `DIRECTORY`. I think we can add those too here.

Reply:
@andrejtonev what do you think on this one?

Reply:
Ah sorry, it slipped my mind. I'll add them now.